pkg/controllers/management/auth/common/common.go

```diff
@@ -0,0 +1,10 @@
+package common
+
+import "github.com/rancher/norman/httperror"
+
+func IsClusterUnavailable(err error) bool {
+	if apiError, ok := err.(*httperror.APIError); ok {
+		return apiError.Code == httperror.ClusterUnavailable
+	}
```

Comment:
This code has existed for a while now and the functionality that it is used for isn't being changed in this PR. I'd prefer not to make this change because that would require me to regression test that functionality.

Reply:
You don't need to change it since you did not edit the code; you just moved it.